src/dateutils.ts

```diff
@@ -20,7 +20,20 @@ export function sameDate(a?: XDate, b?: XDate) {
     return false;
   } else {
     return a?.getFullYear() === b?.getFullYear() && a?.getMonth() === b?.getMonth() && a?.getDate() === b?.getDate();
-  } 
+  }
+}
+
+export function sameWeekRange(a: string, b: string, numberOfDays: number, firstDateInWeek: string){
```

Comment:
I would probably name this method `sameDayRange` and the last pharam 'firstDateInRange'